test/stache-test.js

```diff
@@ -6376,7 +6376,7 @@ function makeTest(name, doc, mutation) {
 	testHelpers.dev.devOnlyTest("Arrays warn about escaping (#600)", 3, function () {
 
 		var map = new SimpleMap({
-			foo: ["<p></p>"]
+			foo: new DefineList(["<p></p>"])
```

Comment:
It seems like #600 was specifically about Arrays, although I can't tell for sure, so I am not sure what is right here. I'm going to dismiss my review, but I think you should wait for @justinbmeyer to weigh in before merging.

Reply:
Ok thanks

Reply:
Yeah, I suppose Array and Objects are built-ins.   Maybe the solution here is to do nothing until a Major release and then remove this warning.  Or we should tell people to call `obj.toString()` in the warning.